src/services/web3/transactions/concerns/transaction.concern.ts

```diff
@@ -11,6 +14,10 @@ export class TransactionConcern {
   ) {}
 
   public shouldLogFailure(error): boolean {
-    return this.config.env.APP_ENV !== 'development' && !isUserError(error);
+    return (
+      this.config.env.APP_ENV !== 'development' &&
+      !isUserError(error) &&
+      !isFaucetRefillError(error)
```

Comment:
Instead of adding multiple conditions here we should have a single catchall conditional like `shouldCaptureError` or similar.

Reply:
isn't `shouldLogFailure` actually that `single catchall conditional`?

Reply:
think he means merge `isUserError` with `isFaucetRefillError` and call it `shouldCaptureError`

Reply:
if so, I don't understand the logic, what for we need one more function here?

Reply:
Sorry, I missed the context of the function these conditionals are in, this `shouldLogFailure` function is essentially what I'm asking for, but it's in the wrong place. We should have a global `shouldCaptureError` function in something like `useErrors` or the utils file Daniel created. We need a single source of truth for error handling.

Reply:
yeah, but I did it as hf to prevent failing errors in sentry, and Daniel did that pr in dev. do you think it's better to do this in dev branch?

Reply:
Yeah fair enough, lets merge this and continue work on global handling.